File: lab/javascript/matematica_e_operadores/positivo_negativo_ou_zero.js
function numberIs(number){
    if (number === 0){
        return "Zero";
    } else if (number > 0){
        return "Positive";
    } else {
        return "Negative";
    }
}


for (let i = -10; i < 11; i++){
    console.log(`${i} = ${numberIs(i)}`);
    console.log();
}

File: lab/python/matematica_e_operadores/positivo_negativo_ou_zero.py
def number_is(number):
    if number == 0:
        return "Zero"
    elif number > 0:
        return "Positive"
    else:
        return "Negative"


numeros = list(range(-10, 11))
for numero in numeros:
    print(f"{numero} = {number_is(numero)}")
    print()
